easy_thumbnails/templatetags/thumbnail.py

```diff
@@ -113,7 +113,10 @@ def render(self, context):
                 return self.bail_out(context)
 
         try:
-            thumbnail = get_thumbnailer(source).get_thumbnail(opts)
+            if settings.THUMBNAIL_PREVENT_GIF_CONVERT and '.gif' in source.url:
```

Comment:
I would rather use
```suggestion
            bits = os.path.splitext(source.url)
            if settings.THUMBNAIL_PREVENT_GIF_CONVERT and bits[1] == '.gif':
```
to explicitly state what's the intention.